Bottom: 3f4d353c7d1fb68ba50c99f0a593c08a1f3a830f
Top:    885a886433bd704c8a35630ef2947a798a36e322
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-05 21:37:40 +0100

Refresh of continue-tinkering-with-3d

---

diff --git a/src/hid/gtk/object3d.c b/src/hid/gtk/object3d.c
index 15fe179..1e2c7e3 100644
--- a/src/hid/gtk/object3d.c
+++ b/src/hid/gtk/object3d.c
@@ -823,10 +823,10 @@ object3d_from_board_outline (void)
 
       edge_info_set_round (UNDIR_DATA (edges[i]),
                            COORD_TO_MM (ct->cx), COORD_TO_MM (ct->cy), -COORD_TO_MM (HACK_BOARD_THICKNESS), /* Center of circle */
-                           0., 0., -1., /* Normal */ COORD_TO_MM (ct->radius)); /* NORMAL POINTING TO -VE Z MAKES CIRCLE CLOCKWISE (??) */
+                           0., 0., -1., /* Normal */ COORD_TO_MM (ct->radius)); /* NORMAL POINTING TO -VE Z MAKES CIRCLE CLOCKWISE */
       edge_info_set_round (UNDIR_DATA (edges[npoints + i]),
                            COORD_TO_MM (ct->cx), COORD_TO_MM (ct->cy), 0., /* Center of circle */
-                           0., 0., -1., /* Normal */ COORD_TO_MM (ct->radius));
+                           0., 0., -1., /* Normal */ COORD_TO_MM (ct->radius)); /* NORMAL POINTING TO -VE Z MAKES CIRCLE CLOCKWISE */
       edge_info_set_stitch (UNDIR_DATA (edges[2 * npoints + i]));
     }
